src/hooks/useSelectionState/useSelectionState.stories.tsx

```diff
@@ -106,3 +106,52 @@ export const ExpandableTable: React.FunctionComponent = () => {
     </Table>
   );
 };
+
+export const ExternalState: React.FunctionComponent = () => {
+  // import { Checkbox } from '@patternfly/react-core';
+
+  interface IFruit {
+    name: string;
+    // Anything else, this can come straight from API data
```

Comment:
Maybe add Example along the comment?

Reply:
Do you mean just additional unused fields? I'm just trying to demonstrate that it can be any TS type and not necessarily just a string for each item.